Sources/MapboxNavigation/NavigationMapView+BuildingHighlighting.swift

```diff
@@ -90,18 +95,15 @@ extension NavigationMapView {
             highlightedBuildingsLayer.source = "composite"
             highlightedBuildingsLayer.sourceLayer = "building"
             
-            let extrudeExpression: Expression = Exp(.eq) {
-                Exp(.get) {
-                    "extrude"
-                }
-                "true"
-            }
-            
             if extrudeAll {
-                highlightedBuildingsLayer.filter = extrudeExpression
+                highlightedBuildingsLayer.filter = Exp(.eq) {
+                    Exp(.get) {
+                        "extrude"
+                    }
+                    "true"
+                }
             } else {
                 highlightedBuildingsLayer.filter = Exp(.all) {
-                    extrudeExpression
```

Comment:
Some buildings have multiple identifiers, the query result may has `extrusion: false`